src/ooxml/java/org/apache/poi/xssf/usermodel/XSSFTextParagraph.java

```diff
@@ -863,6 +864,41 @@ private boolean fetchParagraphProperty(ParagraphPropertyFetcher visitor){
         return ok;
     }
 
+    public void setTextFillColor(XSSFColor color) {
```

Comment:
could you add a getTextFillColor?

Reply:
Must I return null if no fill colour is found?

Reply:
Probably best to return null if no value is set

Reply:
Ok, I will try to do so. But translating CTColors into XSSFColor may be hard, Is there any helper to do that in the project?

Reply:
XSSFColor has `public static XSSFColor from(CTColor color, IndexedColorMap map)` - you can use `new DefaultIndexedColorMap()`

Reply:
Ok I will try to build `org.openxmlformats.schemas.spreadsheetml.x2006.main.CTColor` from CTSolidColorFillProperties.

I'm wondering if we cannot create a sub interface for all those colors instead of CTColor